### ISSUE TITLE
Strange idle behavior

### ISSUE THREAD

**Issue by desk7** [meyayl/docker-languagetool#47]
Actually I don't know if it is the expected behavior. I run the languagetool docker container in a synology NAS and, while the server is in idle, the log keeps showing

date |  stream | content 
-- | -- | --
2024/11/07 22:35:24 | stdout | 2024-11-07 21:35:24.211 GMT INFO  org.languagetool.server.LanguageToolHttpHandler Handled request in 22ms; sending code 200
2024/11/07 22:35:24 | stdout | 2024-11-07 21:35:24.211 GMT INFO  org.languagetool.server.TextChecker Check done: 13 chars, en-US, requestId: null, #147, null, 0/1 matches, 21ms, agent:-, sent, q:0, h:1, dH:1, r:147, m:all, premium: false
2024/11/07 22:35:24 | stdout | 2024-11-07 21:35:24.189 GMT INFO  org.languagetool.server.LanguageToolHttpHandler Handling POST /v2/check
2024/11/07 22:34:53 | stdout | 2024-11-07 21:34:53.903 GMT INFO  org.languagetool.server.LanguageToolHttpHandler Handled request in 23ms; sending code 200
2024/11/07 22:34:53 | stdout | 2024-11-07 21:34:53.902 GMT INFO  org.languagetool.server.TextChecker Check done: 13 chars, en-US, requestId: null, #146, null, 0/1 matches, 21ms, agent:-, sent, q:0, h:1, dH:1, r:146, m:all, premium: false
2024/11/07 22:34:53 | stdout | 2024-11-07 21:34:53.880 GMT INFO  org.languagetool.server.LanguageToolHttpHandler Handling POST /v2/check
2024/11/07 22:34:23 | stdout | 2024-11-07 21:34:23.576 GMT INFO  org.languagetool.server.LanguageToolHttpHandler Handled request in 22ms; sending code 200

and so on forever.

Is it normal?

My docker-compose.yaml is
```yaml
---
services:
  languagetool:
    image: meyay/languagetool:latest
    container_name: languagetool
    restart: always
    cap_drop:
      - ALL
    cap_add:
      - CAP_SETUID
      - CAP_SETGID
      - CAP_CHOWN
    security_opt:
      - no-new-privileges
    ports:
      - 8010:8010
    environment:
      download_ngrams_for_langs: en
      MAP_UID: 1031
      MAP_GID: 100
    volumes:
      - /volume1/docker/languagetool/ngrams:/ngrams
      - /volume1/docker/languagetool/fasttext:/fasttext
```

**Comment by meyayl** [comment]
>Is it normal?

Let me rephrase that: is this expected default behavior?

Yes it is. Those log entries are caused by the container's health check and are expected.

According the compose reference, you can disable the health check using
```
services:
  languagetool:
    healthcheck:
      disable: true
```
see: https://docs.docker.com/reference/compose-file/services/#healthcheck

Furthermore, you can set the log level using the environment variable `LOG_LEVEL` to `WARN` to make the logs less chatty.
